frontend/lib/map/map/map.dart

```diff
@@ -142,10 +145,40 @@ class _MapState extends State<Map> implements MapController {
   }
 
   void _onMapClick(Point<double> point, clickCoordinates) async {
-    var features = await _controller.queryRenderedFeatures(
-        point, widget.onFeatureClickLayerFilter ?? [], null);
+    var pixelRatio = MediaQuery.of(context).devicePixelRatio;
+
+    var touchTargetSize = pixelRatio * 38.0; // corresponds to 1 cm roughly
```

Comment:
From my testing, for me personally I think 20-25 would be better for iOS.

Reply:
for the touchTargetSize or the cm?

Reply:
I just changed the 38.0 to 20.0 and that worked better for me 😁. Idk what that corresponds to

Reply:
Did you test on a real device or simulator?

Reply:
I tested it on my iPad mini and on my iPhone

Reply:
On thr simulator it also seems too big. On android emulator it seems just fine.

Ill test again on a real android device.
This is very weird because it means that the pixel ratio is wrong..

Reply:
I'll keep it as it is: 1cm

A quick google search also recommended this value